### PR TITLE
[quality_control_stock][8.0][IMP] Package transfer trigger qc

### PR DESCRIPTION
**Pull request by andhit-r**
Now package transfer can trigger QC

### CODE REVIEW

File: quality_control_stock/models/stock_picking.py [hunk 1]
```diff
@@ -46,9 +48,20 @@ def do_transfer(self):
                           'qc.trigger.product_line']:
                 partner = (self.partner_id
                            if qc_trigger.partner_selectable else False)
-                trigger_lines = trigger_lines.union(
-                    self.env[model].get_trigger_line_for_product(
-                        qc_trigger, operation.product_id, partner=partner))
+                if not operation.product_id:
+                    dict_product = package_obj._get_all_products_quantities(
```
Comment (pedrobaeza) [inline quality_control_stock/models/stock_picking.py:52]: Isn't better to just retrieve products with a `mapped` operation over the one2many field of the details of the package? You don't need quantities

**Comment by anajuaristi** [comment]
Hi @andhit-r 
Could you please explain the functionality you want to add? It seems interesting but I can not fully understand it just reading the code. It would be great if you could explain the functional behavior change.

Thank you very much
Ana

**Comment by andhit-r** [comment]
Hi @anajuaristi . 
You can look at this issue https://github.com/OCA/manufacture/issues/133. This PR will add functionality use case describe on that issue. Thanks.

**Comment by anajuaristi** [comment]
Ok @andhit-r. I see. Thank you!!

**Pull request closed without merging**